schemas/IDLs/org.gel.models.report.avro/6.1.0/CommonInterpreted.avdl

```diff
@@ -262,6 +271,10 @@ enum TimeUnit {years, months, weeks, days, hours, minutes, na}
         */
         int modifier;
         /**
+        Activation Strength
+        */
+        ActivationStrength  activationStrength;
```

Comment:
should this be:

`ActivationStrength ActivationStrength;`

not

`ActivationStrength activationStrength;` ?

Reply:
no first one is the type (enumeration) second is the name (camel case starting with lowercase)